src/com/magento/idea/magento2plugin/actions/generation/dialog/NewModuleDialog.java

```diff
@@ -107,6 +107,7 @@
     private final ModuleIndex moduleIndex;
     private final CamelCaseToHyphen camelCaseToHyphen;
     private static final String MAGENTO_BEFORE_DECLARATIVE_SCHEMA_VERSION = "2.2.11";
+    private static final String DEFAULT_MODULE_PREFIX = "module";
```

Comment:
Constants should be declared before properties.
Please, move all constants from here to other constants at the top of this class.